src/components/NoteDetails/NoteDetailsTopBar.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  IconButton,
+  makeStyles,
+  Menu,
+  MenuItem,
+  Typography,
+} from '@material-ui/core';
+import { TopBar } from 'components/TopBar/TopBar';
+import { TopBarLeftButtons } from 'components/TopBar/TopBarLeftButtons';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { useDispatch, useSelector } from 'react-redux';
+import { useRef, useState } from 'react';
+import { TopBarPageTitle } from 'components/TopBar/TopBarPageTitle';
+import { changeParentButtonClickedAction } from 'components/BottomBar/BottomBarActions';
+import { colors } from 'colors';
+import { BackButton } from 'components/BackButton/BackButton';
+
+const useStyles = makeStyles(() => ({
+  moreIcon: {
+    position: 'absolute',
+    color: colors.textColor,
+    right: 0,
+    padding: '12px',
+    marginRight: '40px',
+  },
+}));
+
+const NoteDetailsTopBar = () => {
+  const classes = useStyles();
+  const menuAnchorEl = useRef();
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
+  const selectedNote = useSelector(state => state.notesMindMap.selectedNote);
+  const dispatch = useDispatch();
+
+  const openMenu = () => setIsMenuOpened(true);
+  const closeMenu = () => setIsMenuOpened(false);
+
+  const handleChangeParentButtonClick = () => {
+    dispatch(changeParentButtonClickedAction(selectedNote));
+  };
+
+  const title = selectedNote ? selectedNote.name : 'Add you note';
```

Comment:
I believe we always have a `selecteNote` in Redux. So `title` will never be `'Add you note'`.
I tried creating a new note on your branch and it shows the name of `selectedNote`:

![image](https://user-images.githubusercontent.com/6686679/115889702-d86e8400-a45c-11eb-9485-3fd67de4e0f1.png)

I will create a separate task to fix the title problem when creating a new note.
For now let's just show the `selectedNote.name` as the topbar title.